<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>anaSayfa</class>
 <widget class="QWidget" name="anaSayfa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>236</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>anaSayfa</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>DURUM : </string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QRadioButton" name="tura">
       <property name="text">
        <string>Tura</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QRadioButton" name="yazi">
       <property name="text">
        <string>Yazı</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_1">
       <property name="text">
        <string>PC SEÇİMİ</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>SİZ :              BİLGİSAYAR:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLineEdit" name="label_4">
       <property name="text">
        <string>BİZİM SEÇİMİMİZ:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
